internal/kafka/test/integration/kafkas_test.go

```diff
@@ -527,7 +527,7 @@ func testValidations(h *coreTest.Helper, t *testing.T, kafkaValidations []kafkaV
 	for _, val := range kafkaValidations {
 		errK := test.TestServices.KafkaService.RegisterKafkaJob(val.kafka)
 		if (errK != nil) != val.eCheck.wantErr {
-			t.Errorf("Ung.Expected error %v, wantErr = %v for kafka %s", errK, val.eCheck.wantErr, val.kafka.Name)
+			t.Errorf("Expected error %v, wantErr = %v for kafka %s", errK, val.eCheck.wantErr, val.kafka.Name)
```

Comment:
Unrelated to the intended change in this PR but we could probably use gomega assertion for this?

Reply:
Yes, that's true. I didn't actually see it until you pointed it out. I'll try to change this on this PR as well.

Reply:
done, can you have another look? thanks @JameelB